pkg/eval/json_evaluator_test.go

```diff
@@ -0,0 +1,342 @@
+package eval
```

Comment:
```suggestion
package eval_test
```

A good practice is to have a different package for test, it forces you to test only your public function.